packages/eas-json/schema/eas.schema.json

```diff
@@ -356,12 +366,17 @@
           ]
         },
         "resourceClass": {
-          "description": "The iOS-specific resource class that will be used to run this build. Learn more: https://docs.expo.dev/build-reference/infrastructure/",
-          "markdownDescription": "The iOS-specific resource class that will be used to run this build. [Learn more](https://docs.expo.dev/build-reference/infrastructure/)",
+          "description": "The iOS-specific resource class that will be used to run this build. Learn more: https://docs.expo.dev/build-reference/infrastructure/#ios-build-server-configurations",
+          "markdownDescription": "The iOS-specific resource class that will be used to run this build. [Learn more](https://docs.expo.dev/build-reference/infrastructure/#ios-build-server-configurations)",
           "default": "default",
           "anyOf": [
             {
-              "enum": ["default", "medium", "large", "intel-medium", "m-medium"]
+              "enum": ["default", "medium", "large", "intel-medium", "m-medium"],
+              "markdownEnumDescriptions": [
+                "Maps to \"medium\"",
+                "[Learn more](https://docs.expo.dev/build-reference/infrastructure/#ios-build-server-configurations)",
+                "[Learn more](https://docs.expo.dev/build-reference/infrastructure/#ios-build-server-configurations). Not available on free plan."
```

Comment:
if there is anything that could use some additional description that is not obvious from the name then that would be intel-medium and m-medium, but here you explicitly omitted those values, why?

Reply:
also maybe remove those links, descriptions of resource class field already links to that page, those popups will be hard to read with bunch of urls